### ISSUE TITLE
^ operator gives incorrect results when used with two integer variables

### ISSUE THREAD

**Issue by mansellan** [WaynePhillipsEA/twinbasic#210]
**Describe the bug**
When both operands are any integer type, the `^` (power) operator gives incorrect results (operands are flipped). Integer literals are fine though, even if used with a type suffix.

**To Reproduce**
```vb
    Public Sub Main()

        Dim x As Byte = 5
        Dim y As Long = 4
         
        MsgBox 5& ^ 4& ' 625
        MsgBox x ^ y ' 1024 !!!!
        
        MsgBox 4& ^ 5& ' 1024
        MsgBox y ^ x ' 625 !!!!
	
    End Sub
```

**Expected behavior**
Correct result

**Desktop**
 - OS: Windows 10
 - twinBASIC compiler version 0.10.1389

**Additional context**
If either operand is a floating-point type, the result is correct.

**Comment by WaynePhillipsEA** [comment]
🤔Thankfully with this being a preview, this bug didn't make it into any real app...

Clearly I need to add more test cases around the power operator!!  Thanks, I'll get this sorted.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.1444, thanks for reporting
